src/config.json

```diff
@@ -5,5 +5,6 @@
     "msgRefreshIntervalTime": 5,
     "statusRefreshIntervalTime": 30,
     "eventsRefreshIntervalTime": 20,
-    "volumeRefreshInterval": 10
+    "volumeRefreshInterval": 10,
+    "releaseVersion": "1.6.3"
```

Comment:
@neha can we anyhow import the package.json file to use the release version. I think it will be a headache for us while building the UI package to change it every time. Just a thought!

Reply:
@cloudbehl Agreed. But since the package.json file doesn't get copied to the dist folder and its not a good idea to keep the package manager configuration file in the production environment, I need to make it configurable here. In future, this version number should be available from the API.

Reply:
Good idea to make it configurable for now.